### ISSUE TITLE
Allow a few more commands for chat-commands.js

### ISSUE THREAD

**Issue by D4n2021** [Thomas-Smyth/SquadJS#357]
### What is the issue, e.g. map voting, you would like to solve?
Currently only supports "command", and not "command2", "command3" for example.

### How would you like us to solve this issue?
To make this work like this (example words "version" and "website". Project Reality servers also support those commands iirc)

![grafik](https://github.com/Team-Silver-Sphere/SquadJS/assets/20766261/ffc8d24c-e753-4037-b4e8-33f4bb8b9dec)

### Additional Info
It currently outright ignores command2 and command3 variables.

**Comment by lbzepoqo** [comment]
I will try to submit a PR about this if I finally figure it out how

![image](https://github.com/Team-Silver-Sphere/SquadJS/assets/49586590/8c4df10f-3df8-4f57-aa4d-111acd8aeff0)

If you want to try it, here's what I changed in the `mount` function
```js
  async mount() {
    for (const command of this.options.commands) {
      const commands = Array.isArray(command.command) ? command.command : [command.command];
      for (const cmd of commands) {
        this.server.on(`CHAT_COMMAND:${cmd.toLowerCase()}`, async (data) => {
          if (command.ignoreChats.includes(data.chat)) return;
  
          if (command.type === 'broadcast') {
            await this.server.rcon.broadcast(command.response);
          } else if (command.type === 'warn') {
            await this.server.rcon.warn(data.player.steamID, command.response);
          }
        });
      }
    }
  }
  ```
  
  This way I don't have to reconfigure the existing config and must convert each existing command one I made. As an example:
  ```json
                  {
                    "command": "clear",
                    "type": "broadcast",
                    "response": "Please clear the chat. Thank you!.\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n 󠀡󠀡",
                    "ignoreChats": [
                        "ChatAll",
                        "ChatTeam",
                        "ChatSquad"
                    ]
                },
                {
                    "command": [
                        "nomaincamp",
                        "maincamp",
                        "main",
                        "mainbase",
                        "nomain",
                        "nocamp",
                        "mainbasecamp"
                    ],
                    "type": "broadcast",
                    "response": "Main Camping violates the server rules. Please keep a safe distance from the enemy's main base.",
                    "ignoreChats": [
                        "ChatAll",
                        "ChatTeam",
                        "ChatSquad"
                    ]
                }
```

**Comment by werewolfboy13** [comment]
If not I can attempt to push one for you. All we ask is not to modify flags as they are tied to GitHub actions.

**Comment by lbzepoqo** [comment]
> If not I can attempt to push one for you. All we ask is not to modify flags as they are tied to GitHub actions.

Please do thank you!

**Comment by werewolfboy13** [comment]
> > If not I can attempt to push one for you. All we ask is not to modify flags as they are tied to GitHub actions.
> 
> Please do thank you!

Haven’t forgotten still plan on doing this. Going to make a branch here for it.

**Comment by Thomas-Smyth** [comment]
> ### What is the issue, e.g. map voting, you would like to solve?
> Currently only supports "command", and not "command2", "command3" for example.
> 
> ### How would you like us to solve this issue?
> To make this work like this (example words "version" and "website". Project Reality servers also support those commands iirc)
> 
> ![grafik](https://private-user-images.githubusercontent.com/20766261/327149517-ffc8d24c-e753-4037-b4e8-33f4bb8b9dec.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjEzMTI1MjIsIm5iZiI6MTcyMTMxMjIyMiwicGF0aCI6Ii8yMDc2NjI2MS8zMjcxNDk1MTctZmZjOGQyNGMtZTc1My00MDM3LWI0ZTgtMzNmNGJiOGI5ZGVjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MTglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzE4VDE0MTcwMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWNkZGFkMGQzNWM4YjAyNmFkN2JlMDJlMTBmZDRkOTBiNjM4ZjhiNmYwZjNhNzMwNzFiNmI4NjI1MjI3MzYwZjkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.6WoaXs2DdxC5ivSuIwsuN6St1S6TvxcImByaVofkH_I)
> 
> ### Additional Info
> It currently outright ignores command2 and command3 variables.

I'm not sure I get what is being suggested here. Do you mean you want aliases for the same command? For example, `!web` and `!website` return the same response?

**Comment by lbzepoqo** [comment]
> > ### What is the issue, e.g. map voting, you would like to solve?
> > Currently only supports "command", and not "command2", "command3" for example.
> > ### How would you like us to solve this issue?
> > To make this work like this (example words "version" and "website". Project Reality servers also support those commands iirc)
> > ![grafik](https://private-user-images.githubusercontent.com/20766261/327149517-ffc8d24c-e753-4037-b4e8-33f4bb8b9dec.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjEzMTI1MjIsIm5iZiI6MTcyMTMxMjIyMiwicGF0aCI6Ii8yMDc2NjI2MS8zMjcxNDk1MTctZmZjOGQyNGMtZTc1My00MDM3LWI0ZTgtMzNmNGJiOGI5ZGVjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MTglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzE4VDE0MTcwMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWNkZGFkMGQzNWM4YjAyNmFkN2JlMDJlMTBmZDRkOTBiNjM4ZjhiNmYwZjNhNzMwNzFiNmI4NjI1MjI3MzYwZjkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.6WoaXs2DdxC5ivSuIwsuN6St1S6TvxcImByaVofkH_I)
> > ### Additional Info
> > It currently outright ignores command2 and command3 variables.
> 
> I'm not sure I get what is being suggested here. Do you mean you want aliases for the same command? For example, `!web` and `!website` return the same response?

Now that I see Thomas' comment, would it be better to add another option called `aliases` instead of my suggestion of modifying the `command` option? @werewolfboy13

**Comment by D4n2021** [comment]
> Do you mean you want aliases for the same command? For example, `!web` and `!website` return the same response?

indeed! Back in the days in [PR](https://squad.fandom.com/wiki/Project_Reality) (and still working nowadays afaik), !r, !report and !calladmin worked for the function to send a message to admins.
